cli/src/formatter.rs

```diff
@@ -389,9 +389,36 @@ fn color_for_name(color_name: &str) -> Result<Color, config::ConfigError> {
         "bright magenta" => Ok(Color::Magenta),
         "bright cyan" => Ok(Color::Cyan),
         "bright white" => Ok(Color::White),
-        _ => Err(config::ConfigError::Message(format!(
-            "invalid color: {color_name}"
-        ))),
+        _ => {
+            if name_or_hex.len() == 7 && name_or_hex.starts_with('#') {
+                color_for_hex(name_or_hex)
+            } else {
+                Err(config::ConfigError::Message(format!(
+                    "invalid color: {}",
+                    name_or_hex
+                )))
+            }
+        }
+    }
+}
+
+fn color_for_hex(color: &str) -> Result<Color, config::ConfigError> {
+    if color.len() == 7 && color.starts_with('#') {
+        let r = u8::from_str_radix(&color[1..3], 16);
+        let g = u8::from_str_radix(&color[3..5], 16);
+        let b = u8::from_str_radix(&color[5..7], 16);
+        match (r, g, b) {
+            (Ok(r), Ok(g), Ok(b)) => Ok(Color::Rgb { r, g, b }),
+            _ => Err(config::ConfigError::Message(format!(
+                "invalid color: {}",
```

Comment:
nit: Since this function doesn't provide more detailed error, it can simply return `Option<Color>`.

```rust
let r = ...ok()?;
...
Some(Color::Rgb { r, g, b })
```

Reply:
Returning an Option makes it nicer all around.